packages/cozy-client/src/CozyClient.spec.js

```diff
@@ -1406,6 +1408,8 @@ describe('CozyClient', () => {
           resp = { data: [{ _id: '3', label: 'Rest' }], next: false }
         }
 
+        getRawQueryFromState.mockReturnValue(resp)
```

Comment:
you should add a test for bookmark / offsetBookmark and a test for reading from getRawQueryFromState / offset